src/lib.rs

```diff
@@ -79,6 +79,9 @@ pub use with_alloc::alloc_ringbuffer::AllocRingBuffer;
 #[cfg(feature = "alloc")]
 pub use with_alloc::vecdeque::GrowableAllocRingBuffer;
 
+#[cfg(test)]
+mod conversions;
```

Comment:
Might be nice to just put this in the [`tests` dir](https://doc.rust-lang.org/book/ch11-03-test-organization.html#the-tests-directory)